exercise4/.env.j2

```diff
@@ -0,0 +1,21 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+TRELLO_API_KEY={{trello_api_key}}
+
+TRELLO_API_TOKEN={{trello_api_token}}
+
+TRELLO_BOARD_ID='65ca2d064309d3c8a7902b9b'
+
+TRELLO_TO_DO_LIST_ID='65ca2d064309d3c8a7902ba2'
+
+TRELLO_DOING_LIST_ID='65ca2d064309d3c8a7902ba3'
+
+TRELLO_DONE_LIST_ID='65ca2d064309d3c8a7902ba4'
```

Comment:
Minor, but I'd avoid using `'`s round your env variables - `python-dotenv` is forgiving and handles them without issue, but when we start using Docker that will actually treat them as a part of the literal environment variable, which can be a really annoying discrepancy to find out!